doc/base/classes.xml

```diff
@@ -5395,33 +5395,52 @@
 		</method>
 	</methods>
 	<members>
-		<member name="angular_damp" type="float" setter="set_angular_damp" getter="get_angular_damp" brief="">
+		<member name="angular_damp" type="float" setter="set_angular_damp" getter="get_angular_damp" brief="rate of spin loss">
+      The rate at which objects stop spinning in this area, excluding external forces' influences, measured as a fraction of speed lost per second (1.0 - stops immediately, 0.0 - never stops).
```

Comment:
You have spaces for indent on all member vars description. When you look at the git diff, before you commit, you shouldn't see the `<member` lines in red. Select the whole <members> section and convert the space indents to tabs. You may also want to squash all commits with:
`git rebase -i HEAD~4`

The description is also quite complex. Try to simplify it and chop it up into shorter sentences. Avoid parentheses, as they lengthen sentences. Example:
`The rate at which objects stop spinning in this area. 'angular_damp' represents the amount of speed lost per second. If set to 1.0, physics bodies in the area stop rotating instantly. At 0.0 they never slow down. Doesn't take external forces in account.`

Also avoid passive voice.

Reply:
Done. Will start rebasing in a second.